### ISSUE TITLE
Release fmesher 0.1.1

### ISSUE THREAD

**Issue by finnlindgren** [finnlindgren/fmesher#3]
After #1 completed.

First release:

* [x] `usethis::use_cran_comments()`
* [x] Update (aspirational) install instructions in README
* [x] Proofread `Title:` and `Description:`
* [x] Check that all exported functions have `@return` and `@examples`
* [x] Check that `Authors@R:` includes a copyright holder (role 'cph')
* [x] Check [licensing of included files](https://r-pkgs.org/license.html#sec-code-you-bundle)
* [x] Review <https://github.com/DavisVaughan/extrachecks>

Prepare for release:

* [x] Check if any deprecation processes should be advanced, as described in [Gradual deprecation](https://lifecycle.r-lib.org/articles/communicate.html#gradual-deprecation)
* [x] `usethis::use_version('patch')`
* [x] `git flow release start 0.1.1`
* [x] `usethis::use_github_links()`
* [x] `urlchecker::url_check()`
* [x] `devtools::build_readme()`
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `git flow release publish 0.1.1`

Submit to CRAN:

* [x] `devtools::submit_cran()`
* [x] Approve email

Wait for CRAN...

* [x] Accepted :tada:
* [x] `git flow release finish 0.1.1`
* [x] `git checkout stable`
* [x] `usethis::use_github_release()`
* [x] `git checkout devel`
* [x] If not already done, `usethis::use_dev_version(push = TRUE)`
* [ ] Post on Mastodon
